packages/shared/src/components/icons/Downvote/index.tsx

```diff
@@ -0,0 +1,10 @@
+import React, { ReactElement } from 'react';
+import classNames from 'classnames';
+import UpvoteIcon from '../Upvote';
+import { IconProps } from '../../Icon';
+
+const DownvoteIcon = ({ className, ...rest }: IconProps): ReactElement => (
+  <UpvoteIcon {...rest} className={classNames(className, 'rotate-180')} />
```

Comment:
I was confused why we were using the `UpvoteIcon` then I saw the rotate. Brilliant one ✨